core-interop/build.gradle

```diff
@@ -14,9 +14,11 @@ dependencies {
     api 'org.apache.commons:commons-lang3:3.9'
 
     // <TesterraLogger>
-    api 'org.apache.logging.log4j:log4j-slf4j18-impl:2.13.2'
-    api 'org.apache.logging.log4j:log4j-api:2.13.2'
-    api 'org.apache.logging.log4j:log4j-core:2.13.2'
+    // Fixes Log4Shell vulnerability (https://www.lightbend.com/blog/log4j2-security-advisory)
+    def log4jVersion = "2.15.0"
```

Comment:
Current version is 2.16.0 (https://logging.apache.org/log4j/2.x/changes-report.html#a2.16.0) which do not have that nasty message lookups any more.